src/queries.js

```diff
@@ -20,7 +20,7 @@ function queryAllLabelsByText(
   const matcher = exact ? matches : fuzzyMatches
   const matchNormalizer = makeNormalizer({collapseWhitespace, trim, normalizer})
   return Array.from(container.querySelectorAll('label')).filter(label =>
-    matcher(label.textContent, label, text, matchNormalizer),
+    matcher(label.innerText || label.textContent, label, text, matchNormalizer),
```

Comment:
Actually, I think it may be better to:

1. Change this to: `matcher(getNodeText(label), label, text, matchNormalizer),`
2. Update [getNodeText](https://github.com/ismail-codar/dom-testing-library/blob/fab6064602b2ab5bb35d971bf8a200aa57c32f61/src/get-node-text.js) to try `innerText` before `textContent`

This way we get the benefit of using `innerText` for all text selectors.

Could you make that adjustment here please?

Reply:
I will sent it with a test.